apps/src/templates/rubrics/AiAssessmentFeedbackContext.jsx

```diff
@@ -0,0 +1,5 @@
+import {createContext} from 'react';
+
+const AiAssessmentFeedbackContext = createContext(null);
+
+export default AiAssessmentFeedbackContext;
```

Comment:
This is the central coordination of the feedback context that both the feedback radio (thumbs up/down buttons) and actual feedback component (submit form) use to set the state of the radio buttons.

This was necessary to add because the thumbs up and down buttons needed to be placed in a separate branch of the UI so that they could be rendered in line with the score and temperature widget.